requirements.txt

```diff
@@ -1,6 +1,5 @@
 sqlite3@3.43.2 -DCMAKE_POSITION_INDEPENDENT_CODE=On
 boost@1.83 -DCMAKE_POSITION_INDEPENDENT_CODE=On --build -DCMAKE_CXX_FLAGS=" -std=c++14 -Wno-enum-constexpr-conversion -Wno-deprecated-builtins -Wno-deprecated-declarations "
-ROCmSoftwarePlatform/half@10abd99e7815f0ca5d892f58dd7d15a23b7cf92c --build
```

Comment:
[Q] Where we'll take half after this?

Reply:
How about `apt-get install half` ? :)

Now if we use `amdgpu-install -y --usecase=rocm ` as part of the dockerfile script, both `half` and `composablekernel` will be pre-installed. Then if we try to install it via building dependencies there will be errors.

Reply:
@junliume I recommend keeping this, but commenting this out with `#` (and maybe putting some short explanation). This can be useful for clients who use previous version of ROCm which does not install `half` by default.